package.json

```
@@ -15,7 +15,7 @@
     "coronasafe Contributors"
   ],
   "homepage": "https://care.coronasafe.in",
-  "proxy": "https://careapi.coronasafe.in",
+  "proxy": "http://localhost:9000",
```

```suggestion
  "proxy": "https://careapi.coronasafe.in",
```
Don't forget to replace this : )